src/dndbeyond_character.pyj

```diff
@@ -223,10 +223,21 @@ def rollItem(force_display=False):
                     rite_die = "1d10"
                 damages.append(rite_die)
                 damage_types.append("Crimson Rite")
-        if character.getSetting("ranger-dread-ambusher", False):
-            damages.append("1d8")
-            damage_types.append("Ambush")
-            character.mergeCharacterSettings({"ranger-dread-ambusher": False})
+
+        #Ranger abilities    
+        if "Ranger" in character._classes:
+            rangerlevel = character.getClassLevel("Ranger")

```

Comment:
No need to grab this variable unless it's for Planar Warrior, so better to put it inside the if